crates/atuin-client/src/import/csv_history.rs

```diff
@@ -0,0 +1,105 @@
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+use std::path::PathBuf;
+
+use async_trait::async_trait;
+use csv::Reader;
+use eyre::{Result, eyre};
+use time::OffsetDateTime;
+
+use super::{Importer, Loader};
+use crate::history::History;
+
+const HISTORY_FILE: &str = "history.csv"; // Global constant for history file
+
+#[derive(Debug)]
+pub struct CsvHistoryImporter;
+
+fn default_histpath() -> Result<PathBuf> {
```

Comment:
The `default_histpath` approach made sense for classic shell history import where there is usually an expected location for data, but I think in this case it isn't needed

Instead, we should make it so that the import csv subcommand takes a mandatory argument of the file to import

Reply:
there's not a facility i'm seeing in any of the other importers to specify the location of a file as part of the command line parsing - am i missing it somewhere?

Reply:
trying to dig more on this (excuse my rust n00bness) - it looks like it would be either a modification to the main Importer class, which means everything would get an option, but only one of them would support it, or i create a single "one off" function to basically mimic `async fn import` but takes a path.

are there better options i'm not seeing?

Reply:
@ellie - any suggestions on the above comment?